lib/VersionHistory/VersionViewContext/VersionViewContext.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 import { createContext, useMemo } from 'react';
 import { get } from 'lodash';
 
-import { useVersionsDifference } from '../hooks';
+import { useVersionsDifference } from '../hooks/useVersionsDifference';
```

Comment:
what about index file?

Reply:
imported this way on purpose to avoid circular dependency